pandas/tests/test_nanops.py

```diff
@@ -911,10 +911,11 @@ def test_non_convertable_values(self):
         msg = "Could not convert foo to numeric"
         with pytest.raises(TypeError, match=msg):
             nanops._ensure_numeric("foo")
-        msg = "Could not convert {} to numeric"
+
+        # with the wrong type, python raises TypeError for us
+        msg = "argument must be a string or a number"
```

Comment:
Is a string an allowable value here? Not familiar but most of the other exceptions seem to require numeric

Reply:
"1" or "2.0" or "3+5j" are all acceptable